styles/_package_section.scss

```diff
@@ -1,7 +1,7 @@
 .package-section {
   background: #f5fafe;
 }
-.package__h2 {
+.package-section__h3 {
```

Comment:
Could you please rename it? Don't use h2 h3 or other tags for class naming.

Reply:
renamed